compiler/il/OMRILOps.hpp

```diff
@@ -191,8 +203,8 @@ class ILOpCode
       TR::ILOpCodes opcode = op._opCode;
 
       return (TR::VectorOperation) ((opcode < (TR::NumScalarIlOps + TR::NumOneVectorTypeOps)) ?
-                                   (opcode - TR::NumScalarIlOps) / TR::NumVectorTypes :
-                                    (opcode - TR::NumScalarIlOps - TR::NumOneVectorTypeOps) / (TR::NumVectorTypes * TR::NumVectorTypes));
+                                    (opcode - TR::NumScalarIlOps) / TR::NumVectorTypes :
+                                    (opcode - TR::NumScalarIlOps - TR::NumOneVectorTypeOps) / (TR::NumVectorTypes * TR::NumVectorTypes) + TR::firstTwoTypeVectorOperation - 1);
```

Comment:
@gita-omr 
I think you do not have to subtract 1. At least it is not consistent with `createVectorOpCode`.
I am getting `vconv` for `getVectorOperation(createVectorOpCode(TR::v2m, ...`

Reply:
Yes, I think you are right!